doc/ws.md

```diff
@@ -242,7 +242,8 @@ This class represents a WebSocket. It extends the `EventEmitter`.
 
 - `address` {String|url.URL} The URL to which to connect.
 - `protocols` {String|Array} The list of subprotocols.
-- `options` {Object}
+- `options` {Object} Options given do not have any effect if parsed from the URL
```

Comment:
Would you mind moving this note after "Any other option allowed in `http.request()` or `https.request()`" (the latest element in the list)? Thank you.

Reply:
done